tests/integration/bootstrap.sh

```diff
@@ -48,47 +47,49 @@ setup_otbr() {
 
 setup_openthread() {
     set -e
-    git clone ${OPENTHREAD_CCM_REPO} ${OPENTHREAD_CCM} --branch ${OPENTHREAD_CCM_BRANCH}
+    git clone ${OPENTHREAD_REPO} ${OPENTHREAD} --branch ${OPENTHREAD_BRANCH}
```

Comment:
What if OpenThread updates cause ot-commissioner tests to fail ? Should we use a fixed stable version of openthread? Thoughts? @librasungirl

Reply:
On one hand, currently we don't have a regular stable release process; on the other hand, there is no test against external commissioner in OT's repo, however we may want to ensure the stability of master branch if possible. So it might also be an early warning to ensure the quality of OT if there is fails.
Thoughts?

Reply:
@simonlingoogle @librasungirl Agree that integration tests here can reveal openthread bugs related to external commissioning. We may leave this to future enhancement to get an ideal solution Here let's get this blocking PR merged first. Thoughts?

Reply:
agree